docker/transmart-api-server/docker-entrypoint.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+set -e
+
+# Error message and exit for missing environment variable
+fatal() {
+		cat << EndOfMessage
+###############################################################################
+!!!!!!!!!! FATAL ERROR !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+###############################################################################
+			The variable with the name '$1' is unset.
+			Please specify a value in this container environment using
+			-e in docker run or the environment section in Docker Compose.
+###############################################################################
+EndOfMessage
+		exit 1
+}
+
+# Check that Postgres and Keycloak Host are configured
+# via environment variables
+[ ! -z ${PGHOST+x} ] || fatal 'PGHOST'
+[ ! -z ${PGPORT+x} ] || fatal 'PGPORT'
+[ ! -z ${KEYCLOAK_HOST+x} ] || fatal 'KEYCLOAK_HOST'
+[ ! -z ${KEYCLOAK_PORT+x} ] || fatal 'KEYCLOAK_PORT'
```

Comment:
having one `KEYCLOAK_URL` variable instead of separate ones for host and port seems to be a better option - less variables plus enables referring to keycloak via DNS.
It might be useful to add `KEYCLOAK_CLIENT` and `KEYCLOAK_REALM` variables too, but this is something we could discuss later.